addon/src/components/animated-if.ts

```diff
@@ -1,5 +1,16 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
+import type { AnimatedEachSignature } from './animated-each.ts';
+
+interface AnimatedIfSignature<T> {
```

Comment:
@SergeAstapov is it intentional that this signature doesn't include the positional version of `predicate`? If so, the docs may need to be updated to reflect that, since even the comment directly below this shows `{{#animated-if showThing use=transition}}`

Reply:
@dfreeman it definitely was a miss. Thank you for noticing, I'll create a follow up PR with fix